### PR TITLE
BUILD-249: openshift/library sync for 4.8

### PR DESCRIPTION
**Pull request by gabemontero**
deleted imagestreamtags:
nodejs 10, 10-ubi7
perl 5.26-el7 5.26
postgresql 10-el7 10
ruby 2.5-ubi7 2.5
rhdm-decisioncentral 7.9.0
rhdm-kieserver 7.9.0
rhpam-businesscentral-monitoring 7.9.0
rhpam-businesscentral 7.9.0
rhpam-smartrouter 7.9.0

### CODE REVIEW

**Comment by gabemontero** [comment]
samples operator has been successfully upgrading in the failed e2e-aws-upgrade jobs

those failures according to sippy are https://bugzilla.redhat.com/show_bug.cgi?id=1943334

**Comment by gabemontero** [comment]
`: [sig-network-edge][Conformance][Area:Networking][Feature:Router] The HAProxy router should pass the gRPC interoperability tests [Suite:openshift/conformance/parallel/minimal]`

/test e2e-aws

**Review by adambkaplan — APPROVED** [review]
LGTM

Need docs and QE approvals on this.

**Comment by gabemontero** [comment]
/assign @bparees - need pillar lead application of valid bug since build api is a no feature freeze team
/assign @xiuwang - qe approval as @adambkaplan noted .... passing e2e's are sufficient IMO, but let me know what you think
/assign @rolfedh - docs approval as @adambkaplan noted .... I've listed the removed imagestreamtags we now include in the release notes in the description of this PR

/hold

I'll unhold after everybody chimes in

**Review by rolfedh — APPROVED** [review]
/lgtm

**Comment by gabemontero** [comment]
/assign @adambkaplan 

please go ahead and lgtm .... I'll remove the hold when @xiuwang chimes in

**Review by adambkaplan — APPROVED** [review]
/lgtm

File: assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-runtime-rhel.json
Comment (xiuwang) [inline assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-runtime-rhel.json:76]: 3.1-ubi8 tag is not removed from this pr

3.1-ubi8
  tagged from registry.access.redhat.com/ubi8/dotnet-31:3.1

File: assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-rhel.json
Comment (xiuwang) [inline assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-rhel.json:95]: 3.1-ubi8 tag is not removed from this pr

3.1-ubi8
tagged from registry.access.redhat.com/ubi8/dotnet-31:3.1

File: assets/operator/ocp-x86_64/fis/imagestreams/fuse7-karaf-openshift-rhel7.json
Comment (xiuwang) [inline assets/operator/ocp-x86_64/fis/imagestreams/fuse7-karaf-openshift-rhel7.json:138]: fuse-karaf 1.7 tag is not removed
1.7
  tagged from registry.redhat.io/fuse7/fuse-karaf-openshift:1.7
    prefer registry pullthrough when referencing this tag

  Red Hat Fuse 7.7 Karaf S2I images.
  Tags: builder, jboss-fuse, java, karaf, xpaas, hidden
  Supports: jboss-fuse:7.7.0, java:8, xpaas:1.2

  * registry.redhat.io/fuse7/fuse-karaf-openshift@sha256:4ffd7ccbe8ff0aa2e09e1c8a72410aadc721ed3ed227890f03ce3f8aa2b33700
      33 minutes ago

File: assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-rhel.json
Comment (gabemontero) [reply assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-rhel.json:95]: good catch @xiuwang - thanks !

@rolfedh I'm updating the removed list in this PR accordingly

File: assets/operator/ocp-x86_64/fis/imagestreams/fuse7-karaf-openshift-rhel7.json
Comment (gabemontero) [reply assets/operator/ocp-x86_64/fis/imagestreams/fuse7-karaf-openshift-rhel7.json:138]: again good catch @xiuwang thanks

list in description updated @rolfedh

**Comment by gabemontero** [comment]
I'll still wait @xiuwang for you to apply qe approved before I remove the hold and let this PR merge

File: assets/operator/ocp-x86_64/fis/imagestreams/fuse7-karaf-openshift-rhel7.json
Comment (rolfedh) [reply assets/operator/ocp-x86_64/fis/imagestreams/fuse7-karaf-openshift-rhel7.json:138]: > again good catch @xiuwang thanks
> 
> list in description updated @rolfedh

ack

Comment (rolfedh) [reply assets/operator/ocp-x86_64/fis/imagestreams/fuse7-karaf-openshift-rhel7.json:138]: I've created https://issues.redhat.com/browse/RHDEVDOCS-2931 to track the release notes for this change.

**Pull request merged**
